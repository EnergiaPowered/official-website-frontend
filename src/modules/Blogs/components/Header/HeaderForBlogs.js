import React from "react";
import "./HeaderForBlogs.css";
function HeaderForBlogs() {
    return (
        <section id="header" className="header-section">
            <header className="container">
                <h1 className="header-title">Blogs</h1>
            </header>
        </section>
    );
}
export default HeaderForBlogs;
