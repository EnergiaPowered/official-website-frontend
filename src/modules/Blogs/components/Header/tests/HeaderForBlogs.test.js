import React from "react";
import ReactDOM from "react-dom";
import HeaderForBlogs from "./../HeaderForBlogs";
import { cleanup } from "@testing-library/react";
import { create } from "react-test-renderer";
import Enzyme, { mount } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

afterEach(cleanup);
it("renders without crashing", () => {
    const section = document.createElement("section");
    ReactDOM.render(<HeaderForBlogs />, section);
    ReactDOM.unmountComponentAtNode(section);
});

it("matches snapshot", () => {
    const header = create(<HeaderForBlogs />).toJSON();
    expect(header).toMatchSnapshot();
});
Enzyme.configure({ adapter: new Adapter() });

it("check default title", () => {
    const header = mount(<HeaderForBlogs />);
    expect(header.find("h1").text()).toEqual("Blogs");
    expect(header.toJSON).toMatchSnapshot();
});
