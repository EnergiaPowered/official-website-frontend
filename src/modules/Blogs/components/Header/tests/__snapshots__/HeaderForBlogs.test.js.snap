// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check default title 1`] = `undefined`;

exports[`matches snapshot 1`] = `
<section
  className="header-section"
  id="header"
>
  <header
    className="container"
  >
    <h1
      className="header-title"
    >
      Blogs
    </h1>
  </header>
</section>
`;
